src/main/java/no/training/project/Application.java

```diff
@@ -1,20 +1,25 @@
 package no.training.project;
 
-import no.training.project.resources.HealthResource;
+import no.training.project.resources.datecalculation.DateConversion;
+import no.training.project.resources.health.HealthResource;
+import org.apache.log4j.BasicConfigurator;
 import org.glassfish.grizzly.http.server.HttpServer;
 import org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpServerFactory;
 import org.glassfish.jersey.server.ResourceConfig;
-
 import java.io.IOException;
 import java.net.URI;
 
 public class Application {
 
     public static final URI BASE_URI = URI.create("http://localhost:8080");
 
+
+
     public static void main(String[] args) {
         ResourceConfig config = new ResourceConfig();
         config.register(HealthResource.class);
+        config.register(DateConversion.class);
+
 
          HttpServer httpServer = GrizzlyHttpServerFactory.createHttpServer(BASE_URI, config);
         try {
```

Comment:
it is more natural to block the thread with `Thread.currentThread().join()` and exit with CTRL+C than waiting for any input with `System.in.read()`.

Reply:
Added Thread.currentThread().join() and exiting works with CTRL+C.